src/test/lib/changelog-parser.test.ts

```diff
@@ -149,4 +149,30 @@ describe('ChangelogParser', () => {
     const changelog = changelogParser.parse(changelogText, currentVer);
     assert.equal(changelog.isValid, false);
   });
+
+  it('parses a changelog that follows Conventional Commits convention', () => {
```

Comment:
Sorry for the nitpick, but wondering if the bug was not related to conventional commits. Doesn't this bug get triggered whenever we have a version string more than once? Like this:

```markdown
## 1.3.0 - Upgrading Node.js to 11.3.0
```

If so, how about we say something like "it correctly parses even when a version string appears more than once"? 😃

Reply:
👍 Oh man, you exactly mentioned what I was questioning about too. 

I'll add another test, so one test covers "Conventional Commits" convention, and one for multiple version string headers.

Reply:
PR now includes both tests ✅✅